plugins/uast-kotlin-fir/src/org/jetbrains/uast/kotlin/FirKotlinConverter.kt

```diff
@@ -192,6 +195,25 @@ internal object FirKotlinConverter {
                 is KtImportDirective -> {
                     el<UImportStatement>(build(::FirKotlinUImportStatement))
                 }
+                is LeafPsiElement -> {
+                    when {
+                        element.elementType in identifiersTokens -> {
+                            if (element.elementType != KtTokens.OBJECT_KEYWORD ||
```

Comment:
Can you invert this if condition? Also, shouldn't the second condition be `!= null`?

Reply:
Re: inverting the condition,

This came from the counterpart in FE1.0 UAST, so I prefer to stick to the original form (or to change both sides together).

Re: the 2nd `null` check condition,

IIUC, these lines are read: build `UIdentifier` if the `element` isn't (reserved) `object` keyword, or even if it's `object` keyword, but not from `object` declaration (e.g., escaped with back ticks?).

Anyway, switching to `!= null` shows that all of `object` appeared in the tests are popped as identifers.